exporter/model.go

```diff
@@ -243,6 +244,15 @@ func (r *resource) ImportCommand(ic *importContext) string {
 	return fmt.Sprintf(`terraform import %s%s.%s "%s"`, m, r.Resource, r.Name, r.ID)
 }
 
+var (
+	maxRetries = 5
+	retryDelay = 2
```

Comment:
is this in seconds? 
nit: A small comment regarding time unit would be helpful for readability.